bin/juttle

```diff
@@ -214,6 +217,10 @@ function perform_run(options) {
             program.deactivate();
         }).finally(function() {
             current_program = undefined;
+
+            if (had_an_error) {
+                return Promise.reject();
```

Comment:
What shall we word in the message as we end up printing the message and its like repeating ourselves that there was an error in the program, which was already obvious from the non 0 exit code and various other error lines spewed to the console:

```
rlgomes@x230> ./bin/juttle -e "read http -url 'http://www.google.com/bananas'"
Error: HTTP request failed with 404: Not Found { [RuntimeError: HTTP request failed with 404: Not Found]
  name: 'RuntimeError',
  code: 'HTTP-ERROR',
  info: 
   { status: 404,
     message: 'Not Found',
     body: 'StatusCodeError: 404 - <!DOCTYPE html>\n<html lang=en>\n  <meta charset=utf-8>\n  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">\n  <title>Error 404 (Not Found)!!1</title>\n  <style>\n    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}\n  </style>\n  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>\n  <p><b>404.</b> <ins>That’s an error.</ins>\n  <p>The requested URL <code>/bananas</code> was not found on this server.  <ins>That’s all we know.</ins>\n',
     procName: 'read-http',
     location: { filename: 'main', start: [Object], end: [Object] } } }
└───────────┴───────────┘
Error: Program failed at runtime
```

the `Program failed at runtime` was me doing `reject(Error('Program failed at runtime'));`